### DISCUSSION TITLE
driving_simulation.mp4 파일 관련 오류

### DISCUSSION THREAD

**Discussion by Ish9505** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#85] [2024년도 교육 자료 관련 질문] [answered]
안녕하세요. 18번 강의 6분쯤에서 ros2 파일을 열고 lib 파일 내에 있는 driving_simulation.mp4 파일을 누르면 주행 동영상이 나와야 하는데

저는 영상이 뜨지 않고 사진과 같은 창들이 뜹니다. 해결방법 알려주시면 감사하겠습니다

![스크린샷 2024-07-16 21-07-37](https://github.com/user-attachments/assets/2b3d4b15-4024-4ddf-aada-59e77cfbd305)
![스크린샷 2024-07-16 21-12-04](https://github.com/user-attachments/assets/64fff02a-3f17-4795-94f3-d590c8dbf602)

**Comment by HyeongKeunHong** [comment]
안녕하세요, 

vscode에서 영상 재생하는 것이 아닌, 일반 폴더에 해당 경로에 타고 들어가셔서 동영상 파일을 실행시키면 확인이 가능하실까요?

만일 확인이 가능하시면 그대로 진행하시면 될 것 같습니다.

**Reply by Ish9505** [reply]
![스크린샷 2024-07-16 21-43-15](https://github.com/user-attachments/assets/b99ddb36-9edd-4327-9818-236e965909c0)
일반폴더로 눌러봤는데도 영상이 재생이 안됩니다. 혹시 라벨링 한 데이터 수가 적은것도 원인이 될 수 있을지요.
그렇다면 기존 파일을 지우고 다시 데이터 셋 다운받아봐도 될까요?

**Reply by HyeongKeunHong** [reply]
아니요, 라벨링 한 데이터 수가 적은 것이랑은 무관합니다.
우측 마우스로 속성을 눌렀을 때 어떻게 나오는지 캡처해주실 수 있을까요?

**Reply by Ish9505** [reply]
![스크린샷 2024-07-16 21-55-25](https://github.com/user-attachments/assets/362ba082-8744-470e-97f6-073b955f34c3)
![스크린샷 2024-07-16 21-55-44](https://github.com/user-attachments/assets/e75721d5-ef00-45f4-8aa4-d0173be428ac)
넵. 속성 눌렀더니 이렇게 나옵니다.

**Comment by Jinsun-Lee** [comment]
```
cd ~/ros2_ws/src/camera_perception_pkg/camera_perception_pkg/lib/Collected_Datasets; curl -L -O https://github.com/SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course/raw/main/src/camera_perception_pkg/camera_perception_pkg/lib/Collected_Datasets/driving_simulation.mp4
```
위 명령어 실행하시고, 영상 확인 부탁드립니다.

**Reply by Ish9505** [reply, marked as answer]
![스크린샷 2024-07-16 22-18-22](https://github.com/user-attachments/assets/faddf159-6be8-4ce0-a194-7add6f34b829)
비디오 코덱 다운받아 설치했으며 영상 정상 확인 했습니다. 늦은 밤 감사합니다!
